recipes/geant4/build.sh

```diff
@@ -0,0 +1,31 @@
+#!/bin/bash
+
+declare -a CMAKE_PLATFORM_FLAGS
+if [[ ${HOST} =~ .*darwin.* ]]; then
+	CMAKE_PLATFORM_FLAGS+=(-DCMAKE_OSX_SYSROOT="${CONDA_BUILD_SYSROOT}")
+else
+	CMAKE_PLATFORM_FLAGS+=(-DCMAKE_TOOLCHAIN_FILE="${RECIPE_DIR}/cross-linux.cmake")
+fi
+
+if [[ ${DEBUG_C} == yes ]]; then
+  CMAKE_BUILD_TYPE=Debug
+else
+  CMAKE_BUILD_TYPE=Release
+fi
+
+mkdir geant4-build
+cd geant4-build
+
+cmake                                                          \
+      -DCMAKE_BUILD_TYPE=${CMAKE_BUILD_TYPE}                   \
+      -DCMAKE_INSTALL_PREFIX=${PREFIX}                         \
```

Comment:
I think these options should also be set:
```suggestion
      -DCMAKE_INSTALL_PREFIX=${PREFIX}                         \
      -DGEANT4_BUILD_CXXSTD=17                                 \
      -DGEANT4_USE_SYSTEM_EXPAT=ON                             \
      -DGEANT4_USE_SYSTEM_ZLIB=ON                              \
```
Also what do you think about enabling the graphical components?

Reply:
I would rather keep this recipe _lite_, as Qt drags in half of Anaconda :smile: